stable/openvpn/templates/openvpn-service.yaml

```diff
@@ -12,6 +12,9 @@ spec:
       port: {{ .Values.service.externalPort }}
       targetPort: {{ .Values.service.internalPort }}
       protocol: {{ .Values.openvpn.OVPN_PROTO | upper }}
+{{- if hasKey .Values.service "nodePort" }}
```

Comment:
IMO you should check if .Values.service.type equals NodePort. 

Can you add the nodePort parameter to the values.yaml and add documentation about this to the readme.md and the notes.txt?